src/routes/index.svelte

```diff
@@ -184,6 +184,13 @@
         </p>
       </div>
     </div>
+    <a
+        href="https://leapdao.org/validators/"
```

Comment:
```suggestion
        href="https://leapdao.org/validator"
```
Incorrect link.
You can also add anchors on target page, so it will scroll right down to the form, where one can submit his email

Reply:
yeah thought about the anchor but i think the home page doesn't give enought information so it would be better to arrive on top of the page and be able to read through the core infos about Validations

Reply:
and thanks for the info about the wrong  link